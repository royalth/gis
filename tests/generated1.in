20 25
7 4
20 6
12 16
19 17
11 17
6 13
11 9
20 7
16 10
8 9
15 3
6 9
6 5
9 12
18 1
18 8
3 7
1 14
2 13
2 8
2 18
18 15
19 14
15 8
1 19
