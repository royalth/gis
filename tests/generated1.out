4
1 14 2
1 18 1
1 19 3
2 8 1
2 13 3
2 18 2
3 7 2
3 15 3
4 7 1
5 6 3
6 9 1
6 13 2
6 20 4
7 20 3
8 9 4
8 15 2
8 18 3
9 11 2
9 12 3
10 16 1
11 17 1
12 16 2
14 19 1
15 18 4
17 19 2
